packages/devtools-reps/src/object-inspector/index.js

```diff
@@ -0,0 +1,372 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// @flow
+const {
+  Component,
+  createClass,
+  createFactory,
+  DOM: dom,
+  PropTypes,
+} = require("react");
+
+const Tree = createFactory(require("devtools-components").Tree);
+require("./index.css");
+
+const classnames = require("classnames");
+const Svg = require("../shared/images/Svg");
+const {
+  REPS: { Rep, Grip },
+} = require("../reps/rep");
+const {
+  MODE,
+} = require("../reps/constants");
+
+const {
+  createNode,
+  getChildren,
+  getValue,
+  isDefault,
+  nodeHasProperties,
+  nodeIsFunction,
```

Comment:
mhh, I don't know why eslint does not complain about all those errors.
I'll try to figure this out.